src/routes/SafeAppLandingPage/components/UserSafeSection.tsx

```diff
@@ -0,0 +1,141 @@
+import { ReactElement } from 'react'
+import { useSelector } from 'react-redux'
+import { Title } from '@gnosis.pm/safe-react-components'
+import styled from 'styled-components'
+
+import { userAccountSelector } from 'src/logic/wallets/store/selectors'
+import ConnectButton from 'src/components/ConnectButton'
+import { AddressBookEntry } from 'src/logic/addressBook/model/addressBook'
+import useOwnerSafes from 'src/logic/safe/hooks/useOwnerSafes'
+import useLocalSafes, { LocalSafes } from 'src/logic/safe/hooks/useLocalSafes'
+import { lastViewedSafe } from 'src/logic/currentSession/store/selectors'
+import { addressBookState } from 'src/logic/addressBook/store/selectors'
+import CreateNewSafe from './CreateNewSafe'
+import UseYourSafe from './UseYourSafe'
+
+type UserSafeProps = {
+  safeAppUrl: string
+  availableChains: string[]
+  safeAppChainId: string | null
+}
+
+const UserSafeSection = ({ safeAppUrl, availableChains, safeAppChainId }: UserSafeProps): ReactElement => {
```

Comment:
this component checks if the user has compatible safes (see [`getCompatibleSafes()` function](https://github.com/safe-global/safe-react/pull/3845/files#diff-50e3870a76d070888a36438d0a44a65f71da0b4dbd8e0401914734715fb70c9dR73)) and selects the default compatible safe (see [`getDefaultSafe()` function](https://github.com/safe-global/safe-react/pull/3845/files#diff-50e3870a76d070888a36438d0a44a65f71da0b4dbd8e0401914734715fb70c9dR118) )